akka-docs-new/src/test/scala/tutorial_3/Device.scala

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package tutorial_3
+
+import akka.actor.{ Actor, ActorLogging, Props }
+import tutorial_3.Device.{ ReadTemperature, RecordTemperature, RespondTemperature, TemperatureRecorded }
+import tutorial_3.DeviceManager.{ DeviceRegistered, RequestTrackDevice }
+
+object Device {
+
+  def props(groupId: String, deviceId: String): Props = Props(new Device(groupId, deviceId))
+
+  case class RecordTemperature(requestId: Long, value: Double)
+  case class TemperatureRecorded(requestId: Long)
+
+  case class ReadTemperature(requestId: Long)
+  case class RespondTemperature(requestId: Long, value: Option[Double])
```

Comment:
I need to make names more consistent all around. I don't like the name "Current" though because this is a reply to a particular ID and might not be current.